solutions/fully-configurable/variables.tf

```diff
@@ -8,67 +8,39 @@ variable "ibmcloud_api_key" {
   sensitive   = true
 }
 
-variable "provider_visibility" {
-  description = "Set the visibility value for the IBM terraform provider. Supported values are `public`, `private`, `public-and-private`. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/guides/custom-service-endpoints)."
+variable "existing_resource_group_name" {
   type        = string
-  default     = "private"
-
-  validation {
-    condition     = contains(["public", "private", "public-and-private"], var.provider_visibility)
-    error_message = "Invalid visibility option. Allowed values are 'public', 'private', or 'public-and-private'."
-  }
+  description = "The name of an existing resource group to provision resources in."
```

Comment:
` description = "The name of an existing resource group to provision resource in."`